docs/pascal_voc_user_manual.md

```diff
@@ -0,0 +1,246 @@
+
+Also it is possible to add Pascal VOC dataset and specify task for it, for example:
+
+``` bash
+datum add path -f voc_detection <path/to/dataset/ImageSets/Main/train.txt>
+```
+In addition to `voc_detection`, Datumaro supports `voc_action` (for action classification task), `voc_classification`, `voc_segmentation`, `voc_layout` (for person layout task).
+
+To make sure that the selected dataset has been added to the project, you can run `datum info`
+
+## Export to other formats
+Datumaro supports converting Pascal VOC dataset to [all dataset formats that datumaro supports](../docs/user_manual.md#supported-formats). But the converting will be successful only if the output format supports the type of dataset you want to convert.
+
+There are few ways to convert Pascal VOC dataset to other dataset format:
+
+``` bash
+datum project import -f voc -i <path/to/voc>
+datum export -f coco -o <path/to/output/dir>
+# or
+datum convert -if voc -i <path/to/voc> -f coco -o <path/to/output/dir>
+```
+
+Also it is possible using filters for converting, check user manual for information about filtering:
+
+``` bash
+datum convert -if voc -i <path/to/voc> \
+    -f yolo -o <path/to/output/dir> \
+    --filter-mode FILTER_MODE \
+    --filter '<xpath filter expression>'
+```
+
+Some formats have extra arguments for converting, run `datum export -f FORMAT -- -h` for more information. For example voc_segmentation format has extra argument `--label_map` which get opportunity to load user color map for segmentation mask:
+
+``` bash
+datum export -f voc_segmentation -- --label-map mycolormap.txt
+
+# mycolormap.txt [label : color_rgb : parts : actions]:
+# cat:0,0,255::
+# person:255,0,0:head:
+```
+
+Also you can use original voc format of label map, for example:
+
+``` bash
+datum export -f voc_layout -- --label-map voc
+```
+
+## Import to Pascal VOC
+There are few ways to convert dataset to Pascal VOC format:
+
+``` bash
+# export dataset into Pascal VOC format (classification) from existing project
+datum export -p <path/to/project> -f voc -o <path/to/export/dir> -- --tasks classification
+
+# converting to Pascal VOC format from other format
+datum convert -if imagenet -i <path/to/imagenet/dataset> \
+    -f voc -o <path/to/export/dir> \
+    -- --label_map voc --save-images
+```
+
+Argument `--tasks` allow to specify tasks for export dataset, by default datumaro uses all tasks.
+Argument   `--label_map` allow to define user label map, for example
+
+``` bash
+# mycolormap.txt [label : color_rgb : parts : actions]:
+# cat:0,0,255::
+# person:255,0,0:head:
+datum export -f voc_segmentation -- --label-map mycolormap.txt
+```
+
+Or you can use original Pascal VOC label map:
+
+``` bash
+datum export -f voc_layout -- --label-map voc
+```
+
+## Datumaro functionality
```

Comment:
Try to present this part as a tutorial about solving some practical problems. Try to show and explain why we might need to do this. It will much more interesting and useful than just a list of commands.